checking_script.R

```diff
@@ -0,0 +1,28 @@
+table_funcs <- c(ls(loadNamespace("gt")), ls(loadNamespace("kableExtra")), ls(loadNamespace("DT")), ls(loadNamespace("knitr")))
+table_funcs <- table_funcs[which(table_funcs != "%||%")]
+files_to_check <- list.files(".", full.names = TRUE)
```

Comment:
I had set the directory to just `R` to simplify the search, but I realized we may want search deeper into vignettes, inst examples, etc.?